packages/components/select-v2/src/useSelect.ts

```diff
@@ -486,6 +486,18 @@ const useSelect = (props: ISelectV2Props, emit: SelectEmitFn) => {
     emit(UPDATE_MODEL_EVENT, val)
     emitChange(val)
     states.previousValue = props.multiple ? String(val) : val
+
+    nextTick(() => {
+      if (props.multiple && Array.isArray(props.modelValue)) {
```

Comment:
Use the built-in `isArray` function.
```suggestion
      if (props.multiple && isArray(props.modelValue)) {
```

Reply:
fix